src/build/app/tests.rs

```diff
@@ -61,3 +61,68 @@ fn issue_2090() {
 
     assert!(app.subcommands[0].is_set(AppSettings::DisableVersionFlag));
 }
+
+#[test]
+fn long_help_message() {
```

Comment:
These tests should be in `tests/help.rs` file. And please follow the patterns there. You don't need to do `_build()` or `write_long_help`.

Reply:
I'm sorry. I couldn't find `tests/help.rs`.
I've moved the test to `tests/help.rs`.